views/js/util/providers.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
The name "providers" is a bit weird here, the component is fetching the available tests which has nothing to do with the providers. especially in the local context.

Reply:
Changing to `providers/testsProvider`.